.uberenv_config.json

```diff
@@ -0,0 +1,12 @@
+{
+"package_name" : "care",
+"package_version" : "develop",
+"package_final_phase" : "initconfig",
+"package_source_dir" : "../..",
+"spack_url": "https://github.com/davidbeckingsale/spack.git",
```

Comment:
This URL should be updated?

Reply:
actually, this currently relies on a branch that only exists on davidbeckingsale fork of spack. It's publicly accessible.

Reply:
I'm working on getting it upstreamed...